backend/kernelCI_app/views.py

```diff
@@ -141,3 +141,28 @@ def get(self, request, commit_hash):
         summary = self.create_summary(records)
 
         return JsonResponse({"builds": records, "summary": summary}, safe=False)
+
+
+class BuildDetails(View):
```

Comment:
I think we should extract this to a different files otherwise this will become a clusterfuck